pkgs/development/libraries/gdal/default.nix

```diff
@@ -190,8 +190,10 @@ stdenv.mkDerivation (finalAttrs: {
       ++ darwinDeps
       ++ nonDarwinDeps;
 
+  pythonPath = [ python3 python3.pkgs.numpy ];
+
```

Comment:
Please remove the empty line here to follow the coding style of this file

Reply:
I don't understand this comment, all the attributes are separated by a blank line already so this is the current style.

Reply:
Sorry for nitpicking here. We use blank lines to separate sections of code and inside of section there is no blank like. If you look more closely to the code related to testing there is no blank like between for example `doInstallCheck` and `preCheck`.